api/v1/model_handlers.go

```diff
@@ -336,84 +344,106 @@ func (a *Api) containerPaths(request *restful.Request, response *restful.Respons
 	response.WriteEntity(entities)
 }
 
-// allNodes returns a list of all the available node names in the cluster.
+// makeExternalEntityList converts the result of model's getX methods to the external type.
+func makeExternalEntityList(list []model_api.EntityListEntry) []types.ExternalEntityListEntry {
```

Comment:
An entity is something that holds metrics in the model and can be provided as part of a listing. It 
could be a namespace, a pod, a node, or a container.

An entity could also be the cluster itself, but since there is only one cluster entity it cannot be part of a listing